pkg/minishift/oc/oc.go

```diff
@@ -204,3 +200,29 @@ func flagExist(ocCommandOptions []string, flag string) bool {
 	}
 	return false
 }
+
+func mergeKubeconfigs(configs []*clientcmdapi.Config) *clientcmdapi.Config {
```

Comment:
s/mergeKubeconfigs/mergeKubeConfigs/ as Kube and Config are separate words.

Reply:
@praveenkumar Do you want me to fix it during merge?

Reply:
@LalatenduMohanty go ahead, make sure you also change it where we are using this function.